src/Microsoft.VisualStudio.Services.Agent/HostContext.cs

```diff
@@ -49,8 +49,7 @@ public HostContext(string hostType, string logFile)
                 FileAccess.ReadWrite,
                 FileShare.Read,
                 bufferSize: 4096);
-            TextWriterTraceListener traceListener = new TextWriterTraceListener(logStream);
-            _traceManager = new TraceManager(traceListener, GetService<ISecretMasker>());
+            _traceManager = new TraceManager(new HostTraceListener(logStream), GetService<ISecretMasker>());
```

Comment:
might want to leave a comment, since at some point the HostTraceListener should manage the log file as well, like page the trace log, delete logs older than 1 month. etc.